ras_party/views/respondent_view.py

```diff
@@ -113,16 +113,14 @@ def validate_respondent_claim():
     in client services. There is an argument to use a 403 on invalid claims , but that should be a status on the
     resource not the state of the returned data and so that's stretching the use of http status codes somewhat
     """
-    respondent_id = request.args.get("respondent_id", default="").strip()
-    business_id = request.args.get("business_id", default="").strip()
+    party_uuid = request.args.get("respondent_id")
+    business_id = request.args.get("business_id")
+    survey_id = request.args.get("survey_id")
```

Comment:
Given you've added the survey ID to the API, could you please update openapi.md?